README.md

```diff
@@ -108,11 +108,11 @@ Once the NMODL Framework is installed, you can use the Python parsing API to loa
 
 ```python
 from nmodl import dsl
-import os
 
-expsyn = os.path.join(dsl.example_dir(), "expsyn.mod")
+examples = dsl.list_examples() 
+expsyn = dsl.load_example(examples[-1])
 driver = dsl.NmodlDriver()
-modast = driver.parse_file(expsyn)
+modast = driver.parse_string(expsyn)
```

Comment:
do we need `load_example`? that term sound bit heavy to me. I would expect `list_examples` to return full path of models (mod files) and then just use `parse_file`?

Reply:
The user should not rely on a full path / absolute location of package resources because they could come packaged from a zip file. `pkg_resource` allows dealing with exactly those cases.

Reply:
Agree...but the full path is returned by the package after installation. Am I missing something?

Reply:
Depends. Usually it's in the filesystem and yes there is a function to get/construct the full path. But this method is more portable. Do you see any downside to directly providing the content of the modfile?

Reply:
If user have mod file locally and want to try it, `parse_file` example shown in README is straightforward instead of going via load_examples (I thought ;) )

It's ok, this is minor thing. I thought we could avoid that extra function if not needed.